src/main/java/com/incentives/piggyback/user/controller/UserController.java

```diff
@@ -0,0 +1,67 @@
+package com.incentives.piggyback.user.controller;
+
+import com.incentives.piggyback.user.exception.UserNotFoundException;
+import com.incentives.piggyback.user.model.User;
+import com.incentives.piggyback.user.service.UserService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.validation.Valid;
+import java.net.URI;
+
+@Slf4j
+@RestController
+@RequestMapping("/piggyback-user")
```

Comment:
We need to define a standard and stick to it for all services. Probably empty is fine?

Reply:
okay, will remove it.